metric_learn/itml.py

```diff
@@ -44,24 +45,59 @@ def __init__(self, gamma=1., max_iter=1000, convergence_threshold=1e-3,
 
     convergence_threshold : float, optional
 
-    A0 : (d x d) matrix, optional
-        initial regularization matrix, defaults to identity
+    prior : string or numpy array, optional (default='identity')
+         Initialization of the linear transformation. Possible options are
```

Comment:
not initialization, and not linear transformation

Reply:
thanks, done, I've replaced it by "Initialization of the Mahalanobis matrix"